src/common/utils.ts

```diff
@@ -101,6 +101,8 @@ export const removeEmptyValues = <T>(array: T[]): T[] => array.filter(Boolean);
  */
 export const isProd = process.env.NODE_ENV === 'production';
 
+export const isTest = process.env.NODE_ENV === 'test';
```

Comment:
Oh gosh noooo, i see we did this manual until now 🤦 
https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSHuZxrhDNbyZzn3YZ4g-0ZzIhXMiUpj3LBnQ&s

Reply:
not many places, but yeah, for circuit breaker it makes sense to avoid it in tests